### PR TITLE
Add test for resources

### PR DESCRIPTION
**Pull request by ku00**
I'll write tests for resources!

### CODE REVIEW

**Comment by udzura** [comment]
Wow! It's good.
Seems nice test cases for me.

File: test/yao/resources/test_role.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'pry'
```
Comment (hsbt) [inline test/yao/resources/test_role.rb]: Please remove this.

**Comment by hsbt** [comment]
It seems ok without my comment.

**Comment by ku00** [comment]
Thank you very much for review!!

**Comment by hsbt** [comment]
:octocat:

**Pull request merged**